src/prototype_creep_mineral.js

```diff
@@ -513,11 +521,49 @@ Creep.prototype.checkTerminal = function() {
   return false;
 };
 
+/**
+ * check storage for resouces witch dont belon there
```

Comment:
`check storage for resouces which don't belong there

Reply:
ty